code/pizzabot.js

```diff
@@ -1,7 +1,40 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const A = "Vegetarian Pizza"
+const B = "Hawaiian Pizza"
+const C = "Pepperoni Pizza"
+ 
+let pizzaPrice = 80
 
-const pizzaPrice = 80
+ 
+const greeting = () => {
+  const welcome = (`Hey, happy to serve your pizza. We have ${A}, ${B}, ${C}, which one would you like?`)
+  document.getElementById("welcomePizza").innerHTML = welcome
+}
+greeting()
 
-//Put your Javscript code here:
+const amount = () => {
+  const orderQuantity = 'How many pizzas would you like?'
+  document.getElementById("pizzaAmount").innerHTML = orderQuantity
+}
+amount()
+
+
+const checkOrderName = () => {
+  orderInput = document.getElementById("pizza").value
+  numberPizza = document.getElementById("numberPizza").value
+  let totalprice = pizzaPrice * numberPizza
+  if (orderInput === A || orderInput === B || orderInput === C) {
+    if (numberPizza <= 2) {
+      time = '10 min'
```

Comment:
Time is a number, which is a int. So you can instead add "min" to the line instead `${time} min` since it is something that needs to be added to every deliver time!
```suggestion
      time = 10
```